### PR TITLE
Setup CodeWhisperer path in code coverage tool

### PR DESCRIPTION
**Pull request by andrewyuq**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist
- [ ] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt [hunk 1]
```diff
@@ -448,7 +448,6 @@ class CodeWhispererService {
     }
 
     private fun checkRecommendationsValidity(states: InvocationContext, showHint: Boolean): Boolean {
-        val userInput = states.recommendationContext.userInputSinceInvocation
```
Comment (rli) [inline jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt:451]: revert?

File: codecov.yml [hunk 2]
```diff
@@ -41,3 +48,8 @@ ignore:
   - "sdk-codegen/**/*"
   - "jetbrains-rider/**/*.Generated.kt"
   - "**/TelemetryDefinitions.kt"
+
+flags:
+  codewhisperer:
+    paths:
+      - jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/
```
Comment (rli) [inline codecov.yml:55]: what about jetbrains-ultimate?

**Review by rli — APPROVED** [review]

File: codecov.yml (hunk 2 above)
Comment (andrewyuq) [reply codecov.yml:55]: I will do **/src/...

File: jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt (hunk 1 above)
Comment (andrewyuq) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt:451]: This is actually an unused variable but can revert for the sake of this PR

Comment (rli) [reply jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/service/CodeWhispererService.kt:451]: can leave it in, wasn't sure if intentional

**Review by Will-ShaoHua — APPROVED** [review]

**Pull request merged**